### PR TITLE
10 7959f 1 intro confirmation

### PR DESCRIPTION
**Pull request by reiting**
## Summary

Updates form 10-7959f-1 to have form-specific introduction and confirmation pages

- Updating these files caused some errors in other files that were fixed. 

## Related issue(s)

- https://github.com/department-of-veterans-affairs/va.gov-team/issues/76792

## Testing done

- Manual Testing
- Ran yarn test:coverage-app ivc/champva/10-7959f-1 to verify existing tests run and pass

## Screenshots
Top of intro page (it's too long for one screenshot)
Screenshot 2024-02-21 at 2 29 56 PM (3)
Bottom of intro page
Screenshot 2024-02-21 at 2 30 39 PM
Confirmation page
Screenshot 2024-02-21 at 2 29 31 PM

## What areas of the site does it impact?

This form only

### Quality Assurance & Testing

- [x] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [x] Linting warnings have been addressed
- [ ] Documentation has been updated ([link to documentation](#) \*if necessary)
- [x] Screenshot of the developed feature is added
- [ ] [Accessibility testing](https://depo-platform-documentation.scrollhelp.site/developer-docs/wcag-2-1-success-criteria-and-foundational-testing) has been performed

### Error Handling

- [x] Browser console contains no warnings or errors.
- [x] Events are being sent to the appropriate logging solution
- [ ] Feature/bug has a monitor built into Datadog or Grafana (if applicable)

### Authentication

- [ ] Did you login to a local build and verify all authenticated routes work as expected with a test user

### :warning: Team Sites (only applies to modifications made to the VA.gov header) :warning:

- [ ] The vets-website header does not contain any web-components
- [ ] I used the [proxy-rewrite steps](https://github.com/department-of-veterans-affairs/vets-website/tree/main/src/applications/proxy-rewrite#local-dev) to test the injected header scenario
- [ ] I reached out in the `#sitewide-public-websites` Slack channel for questions

## Requested Feedback
N/A

### CODE REVIEW

File: src/applications/ivc-champva/10-7959f-1/helpers/signatureKeyName.js
Comment (michaelclement) [inline src/applications/ivc-champva/10-7959f-1/helpers/signatureKeyName.js:1]: I think this function may break the signature box unless there's a field in the config named `certifierName`. If all we need is to collect a signature at the end and it doesn't have to be paired to a particular applicant or sponsor, you can probably drop this entirely. Just depends on what needs to happen

File: src/applications/ivc-champva/10-7959f-1/config/form.js [hunk 1]
```diff
@@ -36,55 +32,71 @@ const formConfig = {
...
   introduction: IntroductionPage,
   confirmation: ConfirmationPage,
+  v3SegmentedProgressBar: true,
+  preSubmitInfo: {
+    statementOfTruth: {
+      body:
+        'I confirm that the identifying information in this form is accurate and has been represented correctly.',
+      messageAriaDescribedby:
+        'I confirm that the identifying information in this form is accurate and has been represented correctly.',
+      fullNamePath: formData => getNameKeyForSignature(formData),
```
Comment (michaelclement) [inline src/applications/ivc-champva/10-7959f-1/config/form.js:42]: I believe this custom name path function can be left out if the signature doesn't need to match any text values elsewhere in the form (see other comment).

**Review by michaelclement — COMMENTED** [review]
Looks good! Just had a couple thoughts about the signature field that may need to be double-checked

File: src/applications/ivc-champva/10-7959f-1/helpers/signatureKeyName.js
Comment (reiting) [reply src/applications/ivc-champva/10-7959f-1/helpers/signatureKeyName.js:1]: You're right! I wasn't even thinking about this but at least as far as we know, they just have to sign, not match to anyone specifically.

File: src/applications/ivc-champva/10-7959f-1/config/form.js (hunk 1 above)
Comment (michaelclement) [reply src/applications/ivc-champva/10-7959f-1/config/form.js]: Example without the `fullNamePath` property

```
preSubmitInfo: {
    statementOfTruth: {
      body:
        'I confirm that the identifying information in this form is accurate and has been represented correctly.',
      messageAriaDescribedby:
        'I confirm that the identifying information in this form is accurate and has been represented correctly.',
    }
}
```

**Review by michaelclement — APPROVED** [review]
lgtm!

**Review by cloudmagic80 — APPROVED** [review]

**Review by micahchiang — APPROVED** [review]

**Pull request merged**